### PR TITLE
Simplify the getting started flow

### PR DESCRIPTION
**Pull request by davydog187**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -57,6 +57,7 @@ defmodule SurfaceSite.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
+      setup: ["deps.get", "ecto.setup", "cmd npm install --prefix assets"],
```
Comment (msaraiva) [inline mix.exs:60]: I just realised we don't even have a database. Even the `SurfaceSite.Repo` entry is commented out in `application.ex`: https://github.com/surface-ui/surface_site/blob/ff91be2d53e0ce8629e353cb29397ec28a300f44/lib/surface_site/application.ex#L12

I wonder if we really need `ecto.setup`. Probably not, I guess.

Comment (davydog187) [reply mix.exs:60]: want me to remove as part of this PR?

Comment (msaraiva) [reply mix.exs:60]: If you can confirm we don't need it, feel free to do it as part of this PR, otherwise, we can check that later.

Comment (davydog187) [reply mix.exs]: done. Ecto remains as a dependency for one of the examples

**Comment by Malian** [comment]
@davydog187 Due to the release of Surface `v0.4.0` we should rebase this PR to take into account the last changes in the mix.lock file, then we will be able to merge it! Could you do that? Thanks for having simplified the repo :)

**Comment by davydog187** [comment]
@Malian i rebased

**Comment by msaraiva** [comment]
@davydog187 sorry. I missed this one. I should've merged it before.

Thanks! ❤️

**Pull request merged**